chips/src/main/java/com/decathlon/vitamin/compose/chips/VitaminChipRipples.kt

```diff
@@ -0,0 +1,26 @@
+package com.decathlon.vitamin.compose.chips
+
+import android.os.Build
+import androidx.compose.material.ripple.RippleAlpha
+import androidx.compose.material.ripple.RippleTheme
+import androidx.compose.runtime.Composable
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+// todo to delete
```

Comment:
To delete ?

Reply:
Yes, I removed custom Ripple class to use VitaminTheme one